package.json

```diff
@@ -16,9 +16,7 @@
     "format": "prettier --write",
     "lint-ts": "tsc --noEmit --skipLibCheck && eslint --ext .json,.js,.ts,.tsx .",
     "lint-css": "stylelint \"src/**/*.css\"",
-    "link-check": "./scripts/link-check-git-all.sh",
```

Comment:
do we account for all three commands with this change? do we need all of them?

Reply:
I didn't see where this was used and thought it was legacy, but I'll replicate all the old functionality now that I know!